doc/sdk-cli-reference.md

```diff
@@ -143,6 +143,34 @@ pkg/apis/app/v1alpha1/
 └── zz_generated.deepcopy.go
 ```
 
+### openapi
+
+Runs the [kube-openapi][openapi-code-generator] OpenAPIv3 code generator for all Custom Resource Definition (CRD) API tagged fields under `pkg/apis/...`.
```

Comment:
Add the reference marker for `[openapi-code-generator]` at the bottom of this doc.